Algorithms.Tests/Search/InterpolationSearchTest.cs

```diff
@@ -0,0 +1,65 @@
+﻿using NUnit.Framework;
+
+namespace Algorithms.Tests.Search;
+
+public class InterpolationSearchTest
```

Comment:
What are these methods testing if you're not calling the Search function?

Reply:
Implemented the search in the test.